pkg/frontend/querymiddleware/roundtrip.go

```diff
@@ -28,16 +29,21 @@ import (
 
 const (
 	day                              = 24 * time.Hour
-	queryRangePathSuffix             = "/query_range"
-	instantQueryPathSuffix           = "/query"
-	cardinalityLabelNamesPathSuffix  = "/cardinality/label_names"
-	cardinalityLabelValuesPathSuffix = "/cardinality/label_values"
+	queryRangePathSuffix             = "/api/v1/query_range"
```

Comment:
Note to reviwers: adding the `/api/v1/` prefix is not a breaking change because these are the prefixes mapped in `pkg/api/api.go`.